samples/drivers/eeprom/spd/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  description: Serial Presence Detect (SPD) EEPROM sample
+  name: spd sample
+common:
+  platform_allow: intel_rpl_s_crb intel_adl_rvp
+  build_only: true
```

Comment:
I would prefer a build-only test added in `tests/drivers/build_all/eeprom/` and change this sample to allow for run-time verification (e.g. by matching the expected sample output using a console harness).